### PR TITLE
Enrichments: Fix group cases count

### PR DESCRIPTION
**Pull request by kalletlak**
Fix https://github.com/cbioportal/cbioportal/issues/8386

### CODE REVIEW

File: src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx [hunk 1]
```diff
@@ -88,6 +88,28 @@ export default class AlterationEnrichmentContainer extends React.Component<
         return this.props.groups[1];
     }
 
+    @computed get group1QueriedCasesCount() {
+        let caseIds: Set<string> = new Set(
```
Comment (alisman) [inline src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx:92]: you use Set only to enforce uniqueness?

File: src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx [hunk 2]
```diff
@@ -88,6 +88,28 @@ export default class AlterationEnrichmentContainer extends React.Component<
...
+            this.group1.samples.map(sample =>
+                this.props.patientLevelEnrichments
+                    ? sample.uniquePatientKey
+                    : sample.uniqueSampleKey
+            )
+        );
+        return caseIds.size;
+    }
+
+    @computed get group2QueriedCasesCount() {
```
Comment (alisman) [inline src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx:102]: it seems confusing that "cases" will sometimes mean samples, right? i mean, if we're looking at sampleLevelENrichment, this count will a count of samples.  i think "case" typically means patient in our system.

Comment (alisman) [reply src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx:102]: i know we're racing to get this done and we already use "case" generically below.  just want to make sure i understand.

**Review by alisman — APPROVED** [review]

File: src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx (hunk 1 above)
Comment (kalletlak) [reply src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx:92]: yes

**Pull request merged**